microsoft-365/security/defender-endpoint/linux-global-exclusions.md

```diff
@@ -0,0 +1,204 @@
+
+> [!NOTE]
+> Apart from real-time protection, Global Exclusions are also applicable to passive mode as sensor events will still be passed to EDR. However they will not be applied during a mdatp custom scan.
+
+![Exclusions checked at every scope ](images/global-exclusions.png)
+
+Global Exclusions can be useful to avoid incorrect detections on files or software that are unique or customized to your organization. They can also be useful for mitigating performance issues caused by Defender for Endpoint on Linux.
+
+> [!WARNING]
+> Defining exclusions lowers the protection offered by Defender for Endpoint on Linux. You should always evaluate the risks that are associated with implementing exclusions, and you should only exclude files that you are confident are not malicious.
+
+## Supported exclusion types
+
+The following table shows the exclusion types supported by Defender for Endpoint on Linux. You can exclude certain files, folders, processes. These exclusions come under the *global* [scope](linux-preferences.md#scopes-of-exclusion-optional).
+
+Exclusion|Definition|Examples
+---|---|---
+File|A specific file identified by the full path|`/var/log/test.log`
+Folder|All files under the specified folder (recursively)|`/var/log/`
+Process|A specific process specified by the full path and all files opened by it|`/bin/cat
+
+> [!IMPORTANT]
+> Global Exclusions does not support wildcard paths.
+
+## How to configure the list of exclusions
+
+### From the manged configuration
+
+For more information, see [Exclusion Settings under set preferences for Defender for Endpoint on Linux](linux-preferences.md#exclusion-setting-preferences-preview)
+
+#### Sample profile
+
+```JSON
+{
+   "exclusionSettings":{
+        "exclusions":[
+           {
+              "$type":"excludedPath",
+              "isDirectory":true,
+              "path":"/home/*/git<EXAMPLE DO NOT USE>",
+              "scopes": [
+                    "epp"
+              ]
+           },
+           {
+              "$type":"excludedPath",
+              "isDirectory":true,
+              "path":"/run<EXAMPLE DO NOT USE>",
+              "scopes": [
+                    "global"
+              ]
+           },
+           {
+              "$type":"excludedPath",
+              "isDirectory":false,
+              "path":"/var/log/system.log<EXAMPLE DO NOT USE><SAME AS APPLYING ONLY GLOBAL>",
+              "scopes": [
+                    "epp", "global"
+              ]
+           },
+           {
+              "$type":"excludedFileExtension",
+              "extension":".pdf<EXAMPLE DO NOT USE>",
+              "scopes": [
+                    "epp"
+              ]
+           },
+           {
+              "$type":"excludedFileName",
+              "name":"/bin/cat<EXAMPLE DO NOT USE><NO SCOPE PROVIDED - GLOBAL CONSIDERED>"
+           }
+        ],
+        "mergePolicy":"admin_only",
+   }
+}
+```
+
+### From the command line
```

Comment:
This section can probably have a short description and a link to the linux-resources.md, and we can put all the commands and examples.

Reply:
Just added separate command options for global, will combine them together in next iteration of doc change